src/vs/workbench/contrib/search/browser/searchWidget.ts

```diff
@@ -357,13 +357,17 @@ export class SearchWidget extends Widget {
 
 		this.showContextToggle = new Toggle({
 			isChecked: false,
-			title: appendKeyBindingLabel(nls.localize('showContext', "Toggle Context Lines"), this.keybindingService.lookupKeybinding(ToggleSearchEditorContextLinesCommandId), this.keybindingService),
-			icon: searchShowContextIcon
+			title: appendKeyBindingLabel(nls.localize('showContext', "Context Lines"), this.keybindingService.lookupKeybinding(ToggleSearchEditorContextLinesCommandId), this.keybindingService),
+			icon: searchShowContextIcon,
+			inputActiveOptionBorder: inputOptions.inputActiveOptionBorder,
+			inputActiveOptionForeground: inputOptions.inputActiveOptionForeground,
+			inputActiveOptionBackground: inputOptions.inputActiveOptionBackground
 		});
-		this._register(this.showContextToggle.onChange(() => this.onContextLinesChanged()));
+		this._register(attachToggleStyler(this.showContextToggle, this.themeService));
+		dom.append(searchInputContainer, this.showContextToggle.domNode);
```

Comment:
Unconditionally rendering this means it will be shown even when `options.showContextToggle` is off, such as in the search view.

Reply:
Thanks. I moved it into the if condition